gcp-cloud-run/secrets.tf

```diff
@@ -39,30 +40,44 @@ resource "google_secret_manager_secret" "admin_api_key" {
   depends_on = [google_project_service.apis["secretmanager.googleapis.com"]]
 }
 
+resource "time_rotating" "admin_api_key" {
+  count = local.pvault_admin_api_key_rotation_enabled ? 1 : 0
+
+  rotation_days = var.pvault_admin_api_key.rotate_days
```

Comment:
What happens if rotating_days is 0?

Reply:
The resource is disabled, there is a check in the [local var](https://github.com/piiano/vault-deployments/pull/44/files#diff-7faa798e5e95c941a602bd4e22f445a29e948b91bf830199c58e6e9c4e0bb92eR7).